### ISSUE TITLE
Clarification on changing incorrect error message

### ISSUE THREAD

**Issue by clarud** [nus-cs2103-AY2425S1/forum#690]
Our error message is now as shown below
![image](https://github.com/user-attachments/assets/6f30813e-fe8c-4491-9f55-abb312c812dc)

However, our intended feature is such that end date and time must be strictly not before start, hence on when they are equal it is intended. Are we allowed to make that change as the error message here is incorrect?

**Comment by damithc** [comment]
Yes, you can change the error message to reflect 'not before' condition instead of the 'after' condition, to match how the feature behaves.
